src/v2/handlers/lex-bot-alias/package.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "v2-lex-bot-alias-handler",
```

Comment:
are we thinking of using yarn across the board? Do we want to do that now and remove the package files/add them to gitignore?

Reply:
Good catch here - the lambda handlers should have `yarn.lock` over `package-lock.json`.